cabal-install/Distribution/Client/Setup.hs

```diff
@@ -436,7 +436,7 @@ globalCommand commands = CommandUI {
          globalLocalRepos (\v flags -> flags { globalLocalRepos = v })
          (reqArg' "DIR" (\x -> toNubList [x]) fromNubList)
 
-      ,option [] ["logs-dir"]
+      ,option [] ["logsdir", "logs-dir"]
```

Comment:
I *suspect* that CI fails because the new variants are now first, should we add them last?

Reply:
@ChShersh it'd be worth a try; but make it a separate commit in this PR so we can easily revert it if necessary